pyproject.toml

```diff
@@ -26,3 +26,5 @@ aiohttp-json-rpc = "0.11.2"
 ujson = "1.35"
 toml = "0.9.4"
 pysha3 = "1.0.2"
+ed25519 = "1.4.0"
+ecdsa = "0.13"
```

Comment:
I'm sorry but why, if we have this? https://cryptography.io/en/latest/hazmat/primitives/asymmetric/ec/

Also, as I can judge from the repository this is a pure Python implementation which would be simply slower.
https://pypi.org/project/ecdsa/

Reply:
@eugene-babichenko because https://cryptography.io/en/latest/hazmat/primitives/asymmetric/ec/, here we read public key only hex uncompressed and must be converted to x and y coordinate. (but we have compressed keys)

Reply:
Did some research on that. The format currently used is produced by `VerifyingKey.to_string` method and is not ASN.1 based so it cannot be supported with `py-cryptography`. For now we will leave it as it is. I will consider this issue.